src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar--month-length--30 calendar--start-day--fri calendar">
```

Comment:
let's put the block name first in class list for convenience and to improve code readability